pkg/build/api/types.go

```diff
@@ -8,7 +8,10 @@ import (
 )
 
 // BuildLabel is the key of a Pod label whose value is the Name of a Build which is run.
-const BuildLabel = "build"
+const (
+	BuildAnnotation = "build"
```

Comment:
Are we settled on openshift.io and not openshift.com ?  Would like to rename my finalizer tokens if that is the case.  

Sent from my iPhone

> On May 11, 2015, at 2:06 PM, Clayton Coleman <notifications@github.com> wrote:
> 
> In pkg/build/api/types.go:
> 
> > @@ -8,7 +8,10 @@ import (
> >  )
> >  
> >  // BuildLabel is the key of a Pod label whose value is the Name of a Build which is run.
> > -const BuildLabel = "build"
> > +const (
> > +	BuildAnnotation = "build"
> Follow @ironcladlou's examples in his most recent pulls. Namespaced is required going forward.
> 
> —
> Reply to this email directly or view it on GitHub.
>